tools/validate-links.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const TapRender = require("@munter/tap-render");
+const spot = require("tap-spot");
+const hyperlink = require("hyperlink");
+
+const tapRenderInstance = new TapRender();
+
+tapRenderInstance.pipe(spot()).pipe(process.stdout);
+
+const skipPatterns = [
+    "https://",
+    "fragment-redirect",
+    "_site/sponsors",
+    "_site/docs"
```

Comment:
```suggestion
    "/docs"
```

Would this work the same? With `_site/docs`, I'm getting errors locally on Windows because filepaths have backslash separators.

```
  ✖ FAIL load _site\docs
  | operator: load
  | expected: 200 _site\docs
  |   actual: ENOENT: no such file or directory, open 'C:\milos\eslint.org\_site\docs'
  |       at: _site\index.html:193:22 <a href="/docs">...</a>
```

Reply:
Yes, it should work as this is present in`href="/docs"`. We can mention `href="/docs"` itself for more clarity. Are you facing errors for `_site/sponsors` too? If yes, we can use [`path.normalize()`](https://nodejs.org/api/path.html#pathnormalizepath) too.

Reply:
You're right, this happens with `_site/sponsors` too.

Reply:
I'll push a fix

Reply:
@mdjermanovic I've pushed an update, I also tried it on a [windows runner on CI](https://github.com/eslint/eslint.org/actions/runs/4258455519/jobs/7409708999), seems to be working as expected. Can you also verify it once?